4/john-wiegley-captions.srt

```diff
@@ -417,7 +419,7 @@ Now if I do 1 plus 2.
 
 96
 00:07:00,700 --> 00:07:02,600
-The Parsers work, it's not re-cursing.
+The Parser is [inaudible], it's not recursive.

```

Comment:
yes, exactly! I did not make out the 'a', I heard something like 'borting' and did not make the connection.